src/themes/base/components/Text/textStyles.ts

```diff
@@ -0,0 +1,68 @@
+import { ComponentStyleFunctionParam, ICSSInJSStyle } from '../../../types'
+import { truncateStyle } from '../../../../styles/customCSS'
+import { TextVariables } from '../../../base/components/Text/textVariables'
+import { TextProps } from '../../../../components/Text/Text'
+
+export default {
+  root: ({
+    props: { atMention, disabled, error, size, weight, important, success, truncated, temporary },
+    variables: v,
+  }: ComponentStyleFunctionParam<TextProps, TextVariables>): ICSSInJSStyle => {
+    return {
+      display: 'inline-block',
+      ...(truncated && truncateStyle),
+      ...(atMention === true && {
+        color: v.atMentionOtherColor,
+      }),
+      ...(atMention === 'me' && {
+        color: v.atMentionMeColor,
+        fontWeight: v.atMentionMeFontWeight,
+      }),
+      ...(disabled && { color: v.disabledColor }),
+      ...(error && { color: v.errorColor }),
+      ...(success && { color: v.successColor }),
+      ...(temporary && { fontStyle: 'italic' }),
+      ...(important && {
+        fontWeight: v.importantWeight,
+        color: v.importantColor,
+      }),
+
+      ...(weight === 'light' && {
```

Comment:
What would be the chance that the weight would differ from one theme to another?